### PR TITLE
Add command for dbconsole

### PR DESCRIPTION
**Pull request by unrealhoang**
**Status:** In Progress
**Taker:** @Linuus

### CODE REVIEW

File: lib/lotus/cli.rb [hunk 1]
```diff
@@ -76,6 +76,18 @@ def new(name = nil)
       end
     end
 
+    desc 'dbconsole', 'start a db console'
+    method_option :help,           aliases: '-h', desc: 'displays the usage method'
+    def dbconsole(app)
+      if options[:help]
+        invoke :help, ['new']
```
Comment (hieuk09) [inline lib/lotus/cli.rb]: I think it should invoke help of `dbconsole` here.

**Comment by Linuus** [comment]
@unrealhoang Are you still working on this or is it ok that I take a stab at it?

We were discussing this here https://github.com/lotus/lotus/issues/123

Cheers!

**Comment by unrealhoang** [comment]
@Linuus It would be great if you take on it. I still not have free time to battle with it yet :(

**Comment by Linuus** [comment]
Ok, I'll have a look at it :)



Ping @joneslee85

**Comment by joneslee85** [comment]
@Linuus @unrealhoang okay, I reassign this to @Linuus

**Comment by jodosha** [comment]
Closing in favor of #152

**Pull request closed without merging**